image_view/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8)
 project(image_view)
 
-find_package(catkin REQUIRED COMPONENTS camera_calibration_parsers cv_bridge dynamic_reconfigure image_transport message_filters message_generation nodelet rosconsole roscpp std_srvs stereo_msgs)
+find_package(catkin REQUIRED COMPONENTS camera_calibration_parsers cv_bridge dynamic_reconfigure gtk2 image_transport message_filters message_generation nodelet rosconsole roscpp std_srvs stereo_msgs)
```

Comment:
`gtk2` is not a `catkin` package but rather a system dependency so this isn't correct. This needs to be a separate

```
find_package(GTK2 REQUIRED)
```

Reply:
I added that and organized so GTK2 is similar to the OpenCV and Boost dependencies.

Reply:
Thank you!